packages/broker/src/plugins/operator/OperatorPlugin.ts

```diff
@@ -19,19 +19,35 @@ import { MaintainTopologyHelper } from './MaintainTopologyHelper'
 import { inspectRandomNode } from './inspectRandomNode'
 import { ContractFacade } from './ContractFacade'
 
-export const DEFAULT_MAX_SPONSORSHIP_IN_WITHDRAW = 20 // max number to loop over before the earnings withdraw tx gets too big and EVM reverts it
-export const DEFAULT_MIN_SPONSORSHIP_EARNINGS_IN_WITHDRAW = 1 // token value, not wei
-
 export interface OperatorPluginConfig {
     operatorContractAddress: string
+    heartbeatUpdateIntervalInMs: number // 10 secs
```

Comment:
Could remove the values from the comments so they won't get out-of-sync with `config.schema.json`

Reply:
Agree'd, will remove.

Reply:
Fixed